src/aws-api-mcp-server/awslabs/aws_api_mcp_server/core/aws/service.py

```diff
@@ -175,6 +179,14 @@ def interpret_command(
     )
 
 
+def _log_successful_execution(response: InterpretationResponse | None):
+    if response and response.error_code is None and response.as_json is not None:
+        logger.info(
```

Comment:
What is the timing on this recursive method? Can we try with a big response to see how much it adds on top?

Reply:
Results look good, so this should not be too concerning:

```
Total processing time: 0.0331 seconds
Number of users processed: 1000
Number of log entries processed: 500
Average time per user: 0.000033 seconds
Average time per log entry: 0.000066 seconds
```